exec/exec.go

```diff
@@ -165,10 +179,10 @@ type ExitErrorWrapper struct {
 	*osexec.ExitError
 }
 
-var _ ExitError = ExitErrorWrapper{}
+var _ ExitError = &ExitErrorWrapper{}
 
 // ExitStatus is part of the ExitError interface.
-func (eew ExitErrorWrapper) ExitStatus() int {
+func (eew *ExitErrorWrapper) ExitStatus() int {
```

Comment:
why is this needed?

Reply:
because the pointer `*ExitErrorWrapper` is the `ExitError`. It passes by reference. previous code also works, but it de-references the pointer and passes into the function by value. Again, not a must-fix. L182 is required though.